dttools/src/trash.c

```diff
@@ -0,0 +1,62 @@
+
+#include "random.h"
+#include "create_dir.h"
+#include "unlink_recursive.h"
+#include "stringtools.h"
+#include "debug.h"
+
+#include <sys/stat.h>
+#include <string.h>
+#include <errno.h>
+
+static char *trashdir = 0;
+
+void trash_setup( const char *dir )
+{
+	trashdir = strdup(dir);
```

Comment:
Rather than keeping a global trashdir, I think `trash_file` and `trash_empty` should have as an argument their `trash_dir`. I can see us in the future trying to set trash directories per task and being confused that only the last trash directory created works.

In any case, the setup should do nothing if the trashdir has been setup once.